geth/params/defaults.go

```diff
@@ -33,7 +33,7 @@ const (
 
 	// APIModules is a list of modules to expose via any type of RPC (HTTP, IPC, in-proc)
 	// we also expose 2 limited personal APIs by overriding them in `api/backend.go`
-	APIModules = "eth,net,web3,shh,shhext"
+	APIModules = "eth,net,web3,shh,shhext,status"
```

Comment:
thats bad idea, we dont want to allow status by default. i suggest to add `--api-modules` similarly to how geth is doing it

or finally make add a config file support :)

Reply:
@dshulyak  If you start `statusd` without `-status`, the `Status` service is not visible. But I'm not sure about this part. Does it mean that by default that module is callable from dapps even if it's not public?

Reply:
as i remember we have now 2 bindings for rpc, public (node.Attach with public apis and this whitelist) and private (original node.Attach).

public rpc available to any dapp, so if status is in the whistelist... but if you say that Status is not enabled by default than it is not a big problem. but i still prefer to pass it explicitly